reana_client/cli/workflow.py

```diff
@@ -68,8 +68,8 @@ def workflow_execution_group(ctx):
     '--format',
     '_filter',
     multiple=True,
-    help='Format output according to column titles or column values '
-         '(case-sensitive). Use `<colum_name>=<columnn_value>` format. For '
+    help='Format output according to column titles or column values. '
+         'Use `<colum_name>=<columnn_value>` format. For '
```

Comment:
Some typos still remain:

- `colum_name -> column_name` (in two places in the codebase)
- `columnn_value -> column_value` (in two places in the codebase)
- `dislpay -> display` (in two places in the codebase)

Also please rebase against latest master.